applications/dashboard/modules/class.pagermodule.php

```diff
@@ -100,9 +105,53 @@ public function __construct($sender = '') {
         $this->_PropertiesDefined = false;
         $this->_Totalled = false;
         $this->_LastOffset = 0;
+
+        $this->webLinking = Gdn::getContainer()->get(WebLinking::class);
+
         parent::__construct($sender);
     }
 
+    /**
+     * Add prev/next rel meta pagination tags to the document head. Also, add Link header to the response.
+     *
+     * @param Gdn_Controller $controller
+     */
+    private function addPageMeta(Gdn_Controller $controller) {
+        static $pending = true;
+
+        // Make sure this only happens once.
+        if ($pending === true) {
+            /** @var HeadModule $head */
+            $head = $controller->Head;
+            $currentPage = pageNumber($this->Offset, $this->Limit);
+
+            if ($currentPage > 1) {
+                $prevHref = $this->pageUrl($currentPage - 1);
+                $head->addTag('meta', [
+                    'rel' => 'prev',
+                    'href' => $prevHref
+                ]);
+                $this->webLinking->addLink('prev', $prevHref);
+            }
+
+            if ($this->hasMorePages()) {
+                $nextHref = $this->pageUrl($currentPage + 1);
+                $head->addTag('meta', [
```

Comment:
According to the doc, it looks like it's supposed to be a `link` https://webmasters.googleblog.com/2011/09/pagination-with-relnext-and-relprev.html